### PR TITLE
Assign team labels for pull requests

### PR DESCRIPTION
**Pull request by peppy**
In theory. https://github.com/marketplace/actions/team-label-action

May have to make labels for it to work, we'll see.

### CODE REVIEW

File: .github/workflows/pr-team-labels.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: equitybee/team-label-action@main
```
Comment (frenzibyte) [inline .github/workflows/pr-team-labels.yml:12]: I *think* it's failing due to specifying `main` as a version here. No idea why it's specified that way in the action's description.
```suggestion
      - uses: equitybee/team-label-action@v1.0.2
```

**Comment by peppy** [comment]
Hmm not sure. I've added a PAT to this repo's secrets with `public_repo` but it's not finding it.

**Comment by bdach** [comment]
It's probably due to whatever's causing [this `Secret source: None` thing](https://github.com/ppy/osu/runs/7998750160?check_suite_focus=true#step:1:31).

It's plausible that the secrets aren't visible because the workflow run is ran from your private fork in which case the secret wouldn't be there. [There are other github threads that point to that as a possible cause.](https://github.com/actions/starter-workflows/issues/785#issuecomment-770195817)

**Comment by peppy** [comment]
Yeah, I thought that may be the case. Probably simplest to merge and see what happens?

**Comment by bdach** [comment]
Well, if this workflow as-is in this PR doesn't work due to what I'm thinking, then it's going to continue not working even after merge, as every PR is submitted from a fork. But I guess we can try merging, and if the worst comes to pass, then we can try the `workflow_run` route if it doesn't...?

**Comment by bdach** [comment]
Also, this is really weird:

image

The test result workflow shouldn't be firing from the labeling...

**Comment by peppy** [comment]
Will just close this for now. No idea how to fix the CI thing from picking up on it and probably not worth the effort to fix.

**Pull request closed without merging**